docs/certificatessigningrequest-metrics.md

```diff
@@ -6,3 +6,4 @@
 | kube_certificatesigningrequest_condition | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt; <br> `condition`=&lt;approved\|denied&gt; | STABLE |
 | kube_certificatesigningrequest_labels | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt;| STABLE |
 | kube_certificatesigningrequest_cert_length | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt;| STABLE |
+| kube_certificatesigningrequest_annotations | Gauge | `certificatesigningrequest`=&lt;certificatesigningrequest-name&gt;| EXPERIMENTAL |
```

Comment:
Why don't you document the labels for all new metrics?
E.g. for ConfigMaps, you documented label `annotation_CONFIGMAP_ANNOTATION`=&lt;CONFIGMAP_ANNOTATION&gt;, but for CertificateSigningRequest, you don't document all labels.

Reply:
Good catch, this should be consistent.